### ISSUE TITLE
[Feature request] keep the selected navigational profile for the future usage

### ISSUE THREAD

**Issue by katerynakocherhina** [osmandapp/Osmand#9372]
When closing the navigation menu, the selected profile is switched to the default. It could be convenient to have an opportunity to keep it selected until the app is closed.

**Comment by vshcherb** [comment]
This is debatable issue cause historically we had issues before this behavior was introduced. 
We should keep in mind that after navigation most users want to switch to normal map view without speed ... etc widgets.

Important: after navigation profile is always switching to DEFAULT profile which is configurable in General Settings. Obviously the issue here that you can't configure 2 or more Default profiles after which it shouldn't switch them.

**Comment by andrzej-gmt** [comment]
I would prefer to use easy accessible buttons to permanently switch current profile. Could you consider to add an option "last selected" to the settings of default profile? Then we would get current mode of switching, when choosing defined profile as default, and easy switching, when choosing "last selected".

**Comment by andrzej-gmt** [comment]
I can't find previous debates, which lead to current state, but now I have a guess, that it was a kind of misunderstanding. My guess is that users could complain, when navigation mode was different than current profile. For example in bicycle profile, navigation mode was set to hiking. Then user forgot about a change and got weird routing at next attempt.

What is reasonable for me is to reset navigation mode after stopping navigation. What we get is reset profile, which kind of clears wrong state too, but IMHO is an overreaction.

**Comment by scaidermern** [comment]
Example for a use-case for which the current behavior is very annoying: During a hiking or cycling trip, the user switches to the corresponding profile. So far no routing involved to save battery, the user just looks on the map to compare the current position against the displayed GPX track. From time to time the user wants to know the remaining distance and time, he/she starts to calculate the remaining route up to the destination, looks at the distance and time and then aborts the routing mode to save battery. Now OsmAnd switches from the foot/cycling profile back to the default profile.

This is a typical use case for me. It is rather annoying to have to switch back to the foot/cycling profile after the last step.

**Comment by ehermann** [comment]
I had the exact same use-case while hiking, and It's annoying specially if you have different poi/map setup on profiles. 
I understand the point of "We should keep in mind that after navigation most users want to switch to normal map view without speed ... etc widgets."
But in this case I think that the most logical solution would be to switch back to the profile the user was just before the navigation started.

**Comment by andrzej-gmt** [comment]
Actually you don't have to calculate a route to get profile switching. Just press navigation icon and then back key. You can do it by accident.

**Comment by vshcherb** [comment]
Yes this scenario indeed doesn't requires switching back to map profile. It is planned to be fixed though probably with some compromises
